application/values.yaml

```diff
@@ -251,6 +251,9 @@ deployment:
     # options:
     # - name: ndots
     #   value: '1'
+  # -- (string) DNS Policy.
+  # @section -- Deployment Parameters
+  dnsPolicy: ClusterFirst
```

Comment:
I would use the kubernetes default by default, by setting this value to an empty string (`""`).